pkg/runner/collector_selector.go

```diff
@@ -13,13 +13,16 @@ import (
 var (
 	OptionalCollectorNames []string
 	RequiredCollectorNames []string
+	OptInCollectorNames    []string
```

Comment:
Not sure I understand from the name what it is. Sounds very similar to `OptionalCollectorNames`

Reply:
OptIn is off by default, Optional is on by default but can be turned off.

Reply:
I can probably take this out as we moved the log collector to on by default

Reply:
Removed :)